geomet_climate/tileindex.py

```diff
@@ -22,7 +22,7 @@
 import os
 
 import click
-from osgeo import ogr, osr
+from osgeo import ogr, osr, gdal
```

Comment:
Can we keep the alphabetical order i.e. `from osgeo import gdal, ogr, osr`

Reply:
also done